wal/wal_test.go

```diff
@@ -259,9 +268,7 @@ func TestWAL_FuzzWriteRead_Live(t *testing.T) {
 			lock.RUnlock()
 			index++
 		}
-		if r.Err() != io.EOF {
-			testutil.Ok(t, r.Err())
```

Comment:
Did this fail before because after the `if` check above `r.Err()` would become `EOF` before it reached here?

Reply:
This particular `r.Err()` check was fine, I just made that `checkErrorEOF` function for each of the places we're checking the error for `LiveReader`. The issue here is that EOF is an error we expect to receive occasionally with the live reader, especially with the added slowness of running the tests in Travis.